app/src/main/java/io/github/droidkaigi/confsched/activity/SearchActivity.java

```diff
@@ -201,6 +206,14 @@ public void afterTextChanged(Editable s) {
         }
     }
 
+    @Override public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
+        if (actionId == EditorInfo.IME_ACTION_SEARCH) {
+            InputMethodManager imm = (InputMethodManager)getSystemService(Context.INPUT_METHOD_SERVICE);
```

Comment:
I see. I guess it's better to use Dagger because it's easier to mock.
But currently we don't have test. So either is fine 👍